Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM ubuntu:20.04
-RUN apt-get update && apt-get upgrade --yes && apt-get install git python3.8 python3-pip --yes
+FROM ubuntu:22.04
+RUN apt-get update && apt-get upgrade --yes && apt-get install git python3.9 python3-pip --yes
 ADD . /code
 WORKDIR /code
 
 # need to use typing-extensions<4 to deal with issue 387 bug: Docker will not build
 # TODO: Figure out a better solution or wait for it to resolve itself.
 RUN pip install typing-extensions==3.10.0.2
-RUN pip install .[recommended-plugins]
+RUN pip install .
+RUN ape plugins install alchemy ens etherscan foundry hardhat infura ledger solidity template tokens trezor vyper
+RUN pip install eth-rlp==0.3.0
```

Comment:
Hmm, this package seems like a stub. Do you know where this dependency is required from?

Reply:
it is required by `eth-account`

Reply:
> maybe we could try adding this to our `setup.py` as a dependency instead

it conflicts with a number of other dependencies of our current pinned version dependencies however so we would need to update the versions for those pins to be compatible as well.